intg-appletv/driver.py

```diff
@@ -337,9 +342,7 @@ async def on_atv_connected(identifier: str) -> None:
 async def on_atv_disconnected(identifier: str) -> None:
     """Handle ATV disconnection."""
     _LOG.debug("Apple TV disconnected: %s", identifier)
-    api.configured_entities.update_attributes(
-        identifier, {media_player.Attributes.STATE: media_player.States.UNAVAILABLE}
-    )
+    api.configured_entities.update_attributes(identifier, {media_player.Attributes.STATE: media_player.States.OFF})
```

Comment:
This changes the logic in the UI:
- State `OFF` means the device is off / standby, but can be switched on.
- State `UNAVAILABLE` means there is no connection to the device and cannot be woken up. 
  The UI blocks the entity from sending commands (since there's no connection).

If pyatv sends a disconnected event when the ATV is in standby, then this causes issues and we have to solve it differently. In case the device is really disconnected & unreachable and we show it as `OFF`, it will always trigger an error when the user wants to send a command.

Do you know what happens when we reconnect to an ATV in standby? Does it wake up the device (not good), or is the connection active again for a while (which would be good).